### PR TITLE
Making a new Secondary Dataset ReserveDMu

### PR DESCRIPTION
**Pull request by jordan-martins**
This PR is constrained at /Configuration/Skimming:

- creates Configuration/Skimming/python/PDWG_ReserveDMu_SD_cff.py to make the HLT path selection for the new proposed SD. It is intended to run on the ParkingDoubleMuonLowMass PD
- creates a test file Configuration/Skimming/test/test_ReserveDMu_SD_cfg.py
- updates Configuration/Skimming/python/Skims_PDWG_cff.py with the desired data tier for the SD
- updates the Configuration/Skimming/python/autoSkim.py to be picked up automatically either in T0 or in further ReRECOS

### CODE REVIEW

**Comment by jordan-martins** [comment]
please test

**Comment by trtomei** [comment]
I guess that path was not available in Run2011 :)

**Comment by jordan-martins** [comment]
please test

**Comment by youyingli** [comment]
@jordan-martins. Thanks for the help. Could this PR need to backport to 13_1_X and 13_0_X for data taking?

**Comment by perrotta** [comment]
@cms-sw/pdmv-l2 you signed the 13_0_X backport: can we suppose that you also agree with this master version, and you can sign it (and the 13_1_X backport as well). Could you please confirm (with a "+1")

@jordan-martins @cms-sw/pdmv-l2 could you please also confirm that these changes will not be effective until explicitly switched on at Tier0? (In other words, it is not dangerous for data taking even if the 13_0_X version is merged in parallel with this master version)

File: Configuration/Skimming/python/PDWG_ReserveDMu_SD_cff.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import FWCore.ParameterSet.Config as cms
+
+#Trigger bit requirement
+import HLTrigger.HLTfilters.hltHighLevel_cfi as hlt
+ReserveDMu = hlt.hltHighLevel.clone()
+ReserveDMu.TriggerResultsTag = cms.InputTag( "TriggerResults", "", "HLT" )
+ReserveDMu.HLTPaths = cms.vstring(
```
Comment (perrotta) [inline Configuration/Skimming/python/PDWG_ReserveDMu_SD_cff.py:7]:
```suggestion
ReserveDMu.TriggerResultsTag = ( "TriggerResults", "", "HLT" )
ReserveDMu.HLTPaths = [
```

File: Configuration/Skimming/python/PDWG_ReserveDMu_SD_cff.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    'HLT_Tau3Mu_Mu7_Mu1_TkMu1_IsoTau15_v*',
+    'HLT_Tau3Mu_Mu7_Mu1_TkMu1_Tau15_Charge1_v*',
+    'HLT_Tau3Mu_Mu7_Mu1_TkMu1_Tau15_v*',
+    'HLT_Trimuon5_3p5_2_Upsilon_Muon_v*',
+    'HLT_TrimuonOpen_5_3p5_2_Upsilon_Muon_v*')
+ReserveDMu.andOr = cms.bool( True )
+# we want to intentionally throw and exception
+# in case it does not match one of the HLT Paths
+# set to False now, switch to on once matrix is updated
+ReserveDMu.throw = cms.bool( False )
```
Comment (perrotta) [inline Configuration/Skimming/python/PDWG_ReserveDMu_SD_cff.py:62]:
```suggestion
ReserveDMu.throw = False
```

**Review by perrotta — COMMENTED** [review]
Please, if you need to update further these files remove the explicit types, which is a safer way to code these configurations (not for this PR)

**Comment by perrotta** [comment]
+1

**Pull request merged**